python/ray/autoscaler/aws/cloudwatch/example-cloudwatch-dashboard-config.json

```diff
@@ -0,0 +1,331 @@
+[
+   {
+      "type":"metric",
+      "x":0,
+      "y":12,
+      "width":12,
+      "height":6,
+      "properties":{
+         "view":"timeSeries",
+         "metrics":[
+            [
+               "{cluster_name}-ray-CWAgent",
+               "cpu_usage_system",
+               "InstanceId",
+               "{instance_id}",
+               "cpu",
+               "cpu-total"
+            ],
+            [
+               ".",
+               "usage_active",
+               ".",
+               ".",
+               ".",
+               "."
+            ],
+            [
+               ".",
+               "usage_user",
+               ".",
+               ".",
+               ".",
+               "."
+            ],
+            [
+               ".",
+               "usage_idle",
+               ".",
+               ".",
+               ".",
+               "."
+            ]
+         ],
+         "region":"{region}",
+         "stat":"Average",
+         "period":60,
+         "title":"CPU Usage",
+         "yAxis":{
+            "left":{
+               "min":0,
+               "max":100
+            }
+         }
+      }
+   },
+   {
+      "type":"metric",
+      "x":0,
+      "y":12,
+      "width":12,
+      "height":6,
+      "properties":{
+         "view":"timeSeries",
+         "metrics":[
+            [
+               "{cluster_name}-ray-CWAgent",
+               "processes_running",
+               "InstanceId",
+               "{instance_id}",
+               {
+                  "label":"Runnng",
+                  "color":"#4b74bd"
+               }
+            ],
+            [
+               ".",
+               "processes_sleeping",
+               ".",
+               ".",
+               {
+                  "label":"Sleeping",
+                  "color":"#69cbe2"
+               }
+            ],
+            [
+               ".",
+               "processes_dead",
+               ".",
+               ".",
+               {
+                  "label":"Dead",
+                  "color":"#b1d28c"
+               }
+            ],
+            [
+               ".",
+               "processes_zombies",
+               ".",
+               ".",
+               {
+                  "label":"Zombies",
+                  "color":"#78574b"
+               }
+            ]
+         ],
```

Comment:
I think we can cut these out of the default dashboard - users can add them back if they're interested in them specifically.